### ISSUE TITLE
Reports data: Investigate why same validation notices query's results differ over time

### ISSUE THREAD

**Issue by lauriemerrell** [cal-itp/data-infra#1333]
While working to publish the March reports, @Nkdiaz and I noticed that re-running the February reports now returns different results than were returned at the time that those reports were published in early March. 

Specifically, for `ITP ID 10`, running the February reports now returns 0 validation notices, but in the [published report](https://reports.calitp.org/gtfs_schedule/2022/02/10/index.html) we included 2 violations. 

AC for this ticket:
- [ ] Understand what has caused this change in behavior (reports code vs. pipeline code vs. data itself somehow?)
- [ ] Document a follow-up ticket to address that underlying cause

**Comment by lauriemerrell** [comment]
In cell `# 4. Feed files being checked for ----`, if you change:

```
sql_on = (lambda lhs, rhs:
                  (lhs.calitp_extracted_at < rhs.date) &
                  (func.coalesce(lhs.calitp_deleted_at, "2099-01-01") >= rhs.date)
        )
```

To:

**Comment by lauriemerrell** [comment]
Urgent issue has been fixed in #1337. This issue will likely be difficult to reproduce with that fix merged. Per meeting with @Nkdiaz, @holly-g, @blakeflei - deferring to future sprint.

**Comment by lauriemerrell** [comment]
Closing as won't do now that reports is migrated to v2.
